public/designs.js

```diff
@@ -33,8 +33,7 @@ function makeList(event){
     clearForm();
 }
 
-function copyDataToAddForm(event){
-    event.preventDefault();
+function copyDataToAddForm(){
```

Comment:
We need 'event' argument in the function. It is used in line 37:
`const clickedRow = event.target.parentElement;`